### ISSUE TITLE
!leaving a game before it starts doesn't clear your mode !vote

### ISSUE THREAD

**Issue by fndax** [lykoss/lykos#486]
```
01:53:19 < dax> !j def
01:53:19 <@lykos> dax has stumbled upon the village after a night of sleepwalking. They raised the number of players to 2.
01:53:19 <@lykos> dax votes for the default game mode.
[...]
01:56:46 <+dax> !leave fooding
01:56:46 <@lykos> dax poked a bear in the eye with a stick. It seems they were no good in a fight. New player count: 1
[...]
02:06:57 < dax> !j def
02:06:58 <@lykos> dax decided to play a game of werewolf. They raised the number of players to 3.
02:06:58 -- lykos: You have already voted for the default game mode.
02:07:03 <+dax> !v
02:07:03 <@lykos> default: 1
02:07:11 < minetester> !j def
02:07:12 <@lykos> minetester was led into the village by a group of wild goats. They raised the number of players to 4.
02:07:12 <@lykos> minetester votes for the default game mode.
02:07:13 <+dax> !leave
02:07:13 <@lykos> dax slipped on a banana peel and died on impact. New player count: 3
02:07:14 < dax> !v
02:07:15 <@lykos> default: 2
```

(there were no other voters apart from me and minetester)

**Comment by skizzerz** [comment]
Fixed in 52bfcc63bde6dae3530c94f46bfb7ed7e53f6bb2 I think

**Comment by skizzerz** [comment]
Not actually fixed yet
